modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -200,5 +200,43 @@ controller-0$ sudo docker restart octavia_worker
 ====
 Depending on your OpenStack environment, Octavia might not support UDP
 listeners, which means there is no support for UDP Services if Kuryr SDN is
-used.
+used. {rh-openstack} version 16 and later supports Octavia.
 ====
+
+
+== The Octavia OVN Driver
+
+Octavia supports multiple provider drivers through the Octavia API.
+
+To see all available Octavia provider drivers, on a command line, enter:
+----
+$ openstack loadbalancer provider list
+----
+
+The result is a list of drivers:
+----
++---------+-------------------------------------------------+
+| name    | description                                     |
++---------+-------------------------------------------------+
+| amphora | The Octavia Amphora driver.                     |
+| octavia | Deprecated alias of the Octavia Amphora driver. |
+| ovn     | Octavia OVN driver.                             |
++---------+-------------------------------------------------+
+----
+
+Beginning with {rh-openstack} version 16, the Octavia OVN provider driver (`ovn`) is supported on
+{product-title} on {rh-openstack} deployments.
+
+`ovn` is an integration driver for the load balancing
+that Octavia and OVN provide. It supports basic load balancing capabilities,
+and is based on OpenFlow rules. The driver is automatically enabled
+in Octavia by Director on deployments that use OVN Neutron ML2.
+
+If `ovn` is enabled, the Amphora provider
+driver remains available as a fallback driver.
+
+Kuryr offers better support for `ovn` than Amphora. If you choose `ovn`:
```

Comment:
I think this is not the right way to put it. Kuryr supports both. But if OVN is available, kuryr will automatically use it, as OVN present some benefits (the above listed) for our use case (OCP on OSP)